packages/doctor-sdk/package.json

```diff
@@ -22,13 +22,33 @@
   "dependencies": {
     "@rsbuild/doctor-utils": "workspace:*",
     "lodash": "^4.17.21",
-    "source-map": "^0.7.4"
+    "source-map": "^0.7.4",
```

Comment:
It is recommended to use `pnpm add` to add dependencies, ensuring that the dependencies are arranged in alphabetical order

Reply:
This diff is main add a comma

Reply:
👌🏻